test/auth_cluster/test_auth_no_quorum.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import asyncio
+import time
+from test.pylib.manager_client import ManagerClient
+import pytest
+from cassandra.auth import PlainTextAuthProvider
+from test.pylib.util import read_barrier, unique_name, wait_for_cql_and_get_hosts
+from cassandra.query import SimpleStatement
+
+"""
+Tests how cluster behaves when lost quorum. Ideally for operations with CL=1 live part of the
+cluster should still work but that's guaranteed only if auth data is replicated everywhere.
+"""
+@pytest.mark.asyncio
+async def test_auth_no_quorum(manager: ManagerClient) -> None:
+    config = {
+        # disable auth cache
+        'permissions_validity_in_ms': 0,
+        'permissions_update_interval_in_ms': 0,
+    }
+    servers = await manager.servers_add(3, config=config)
+
+    cql = manager.get_cql()
+    hosts = await wait_for_cql_and_get_hosts(cql, servers, time.time() + 60)
+    await manager.servers_see_each_other(servers)
+
+    # create users, this is done so that previous auth implementation (with RF=1) fails this test
+    # otherwise it could happen that all users are luckily placed on a single node
+    roles = ["r" + unique_name() for _ in range(10)]
+    for role in roles:
+        # if not exists due to https://github.com/scylladb/python-driver/issues/296
+        cql.execute(f"CREATE ROLE IF NOT EXISTS {role} WITH PASSWORD = '{role}' AND LOGIN = true")
```

Comment:
it may be nice if I need to call it together with asyncio.gather but here I want to execute things one by one so for readability I'd prefer to keep cql.execute, unless I'm missing some benefit of run_async

Reply:
don't get it, so either cql.execute should have the same adjustments implemented or should not be used at all in tests. I don't need to create roles concurrently -  it will only cause internal retry loops.

Reply:
please document that, maybe execute can be patched to throw error otherwise it's impossible to guess what can be used and what not.